engine/runtime/src/test/scala/org/enso/compiler/test/context/SuggestionBuilderTest.scala

```diff
@@ -433,7 +450,10 @@ class SuggestionBuilderTest extends CompilerTest {
               ),
               selfType      = "Unnamed.Test.MyAtom",
               returnType    = "Number",
-              documentation = Some(" My bar")
+              documentation = Some(" My bar"),
+              documentationHtml = Some(
+                "<html><body><div class=\"doc\" style=\"font-size: 13px;\"><div><div class=\"\"><p>My bar</p></div></div></div></body></html>"
```

Comment:
Please make a helper function that hides `<html><body>...` template. Should look like 
```
htmlDoc("<p>My bar</p>")
```